platforms/documentation/docs/src/snippets/bestPractices/avoidResolvingConfigurationsManually/groovy/build.gradle

```diff
@@ -0,0 +1,24 @@
+plugins {
+    id 'java-library'
+}
+
+// tag::resolve-call[]
```

Comment:
I think you should widen the tag here to show more context, otherwise it isn't clear _when_ this resolution is being done.

Reply:
Also, I think the more common mistake would be to do this with _within a task's configure/register call_, not at the top-level of the buildscript.

Reply:
I see that's in the second item.  I think the 2 points this PR is trying to make:
1) Don't resolve during configuration time on purpose
2) There are some APIs that are eager that are easy to use by accident (like +), watch out for these
Are definitely related, but the callouts in the examples are focused on the wrong thing (task dependencies)

Reply:
I've reworked this sample. please re-check.